java/azure-eventhubs/src/main/java/com/microsoft/azure/servicebus/MessageReceiver.java

```diff
@@ -146,19 +138,28 @@ public void run()
 
 				if (workItemTimedout)
 				{
-					synchronized (MessageReceiver.this.flowSync)
+					// workaround to push the sendflow-performative to reactor
```

Comment:
>workaround [](start = 8, length = 10)

Either this is no longer a workaround or we are not doing something correctly. Do we have a separate issue to track this?